cluster-api/src/main/java/com/sequenceiq/cloudbreak/cluster/api/ClusterModificationService.java

```diff
@@ -31,6 +31,8 @@
 
     Map<String, String> gatherInstalledParcels(String stackName);
 
+    void updateHueKnoxProxyhosts(String knoxProxyhosts) throws Exception;
```

Comment:
this is not really  generic. I would prefer updateServiceConfiguration with possible ways to add more service